### ISSUE TITLE
zprint deletes form

### ISSUE THREAD

**Issue by atdixon** [kkinnear/zprint#173]
Yikes! zprint deleted some of our code...

I've tested this with zprint `1.0.2` and `1.1.0`.

When `:list {:respect-nl? true}` is configured, zprint quietly deletes this anonymous `fn` form:

```
;; yikes! this form is deleted!
(fn [x]
  (bar)
   )
```

But incidentally not if the `fn` is named:

```
;; all is good
(fn foo [x]
  (bar)
   )
```

Nor if there is no hanging parens:

```
;; all is good
(fn foo [x]
  (bar))
```

**Comment by kkinnear** [comment]
That's not good.  Thanks for noticing! I'll get right on it.  Looks like we'll have a 1.1.1 real soon now.  There was one very obscure problem where some code went away several years ago, but generally the tests have caught the few problem like this that have occurred.  Fortunately, I'm all set up for another release and not in the middle of anything right now.

**Comment by atdixon** [comment]
Thanks Kim for the update! Will keep an eye out.

**Comment by kkinnear** [comment]
Fixed in 1.1.1.  Thanks again for noticing this and making a small example to replicate it!
